src/globals/grid/grid.html

```diff
@@ -0,0 +1,29 @@
+<div class="bx--grid">
+  <div class="bx--row">
+    <div class="bx--col-s1">.bx--col-s1</div>
```

Comment:
Swap over to the newer class names - bx--col-xs et al. Could you also throw in an example of the responsive grid? i.e., do a `bx--col-xs-12` and a `bx--col-md-6`